### ISSUE TITLE
Merging with xtensor?

### ISSUE THREAD

**Issue by certik** [wdmapp/gtensor#49]
Would it make sense to contribute this to `xtensor` itself, or is it better as a separate library as it is now?

I think @SylvainCorlay, @wolfv and others might be interested in your work.

**Comment by germasch** [comment]
Right, I've been meaning to contact @SylvainCorlay for like a year now, but never got to the point where I thought gtensor is quite far enough along. But it might be a good idea to have a chat some time ;)
